hbase-server/src/main/java/org/apache/hadoop/hbase/tool/BulkLoadHFilesTool.java

```diff
@@ -376,7 +376,7 @@ public void loadHFileQueue(AsyncClusterConnection conn, TableName tableName,
    * atomically.
    * @return empty list if success, list of items to retry on recoverable failure
    */
-  private CompletableFuture<Collection<LoadQueueItem>> tryAtomicRegionLoad(
+  protected CompletableFuture<Collection<LoadQueueItem>> tryAtomicRegionLoad(
```

Comment:
Nit: Since method is now _protected_, let's add @param tags on javadoc description.